src/specVersions/v0.2/RicardianContractProcessorImpl.ts

```diff
@@ -0,0 +1,34 @@
+import * as Handlebars from 'handlebars'
+
+import { RicardianContractProcessorImpl as RCP_0_1 } from '../v0.1/RicardianContractProcessorImpl'
+
+const implVersion = {
+  major: 0,
+  minor: 2,
+}
+
+const ifHasValue = function(this: any, val: any, options: Handlebars.HelperOptions) {
```

Comment:
any reason this code is duplicated from the `ifHasValue` in the helpers file and that function just isn't imported into this file from helpers instead? https://github.com/EOSIO/ricardian-template-toolkit/blob/40d686c7750d9316f90c52b848c8bcbe48992d68/src/specVersions/v0.2/helpers.ts

Reply:
Doh, I had it here and was refactoring it out into the helper then forgot to remove the original.